WindowsServerDocs/administration/Linux-Package-Repository-for-Microsoft-Software.md

```diff
@@ -25,7 +25,13 @@ Microsoft's Linux Software Repository is comprised of multiple sub-repositories:
 
 ## Configuring the repositories
 
-Repositories can be configured automatically by installing the Linux package that applies to your Linux distribution and version. The package will install the repository configuration along with the GPG public key used by tools such as apt/yum/zypper to validate the signed packages and/or repository metadata.
+Repositories can be configured automatically by installing the Linux package that applies to your Linux distribution and version. The package will install the repository configuration along with the GPG public key used by tools such as apt, yum or zypper to validate the signed packages and/or repository metadata.
+
+Note that not all supported distributions are listed here, see the currect supported package repositories at [https://packages.microsoft.com/](https://packages.microsoft.com/) and the instructions for [manual configuration](#manual-configuration) below.
```

Comment:
For readability, I think it will be an improvement to divide the 2 sentence parts into 2 separate sentences:
```suggestion
Note that not all supported distributions are listed here. See the current supported package repositories at [https://packages.microsoft.com/](https://packages.microsoft.com/) and the instructions for [manual configuration](#manual-configuration) below.
```
Also added a typo correction for "currect", which I am fairly certain you meant to write as 'current'.